modules/data.atmosphere/R/extract_local_CMIP5.R

```diff
@@ -40,15 +40,18 @@ extract.local.CMIP5 <- function(outfolder, in.path, start_date, end_date, site_i
   # Some GCMs don't do leap year; we'll have to deal with this separately
   no.leap <- c("bcc-csm1-1", "CCSM4")
   
+  if(scenario == "p1000" | GCM=="MPI-ESM-P") date.origin=as.Date("850-01-01")
+  if(scenario == "historical" & GCM!="MPI-ESM-P") date.origin=as.Date("1850-01-01")
+  
```

Comment:
I think with both no.leap and date.origin, the user should also be able to pass these in optionally as well, assuming new CMIP5 (or other scenarios) or future CMIP6 runs have different configurations. This also makes it clear to someone using this function that these are things to watch out for.